packages/google-config-ui/package.js

```diff
@@ -0,0 +1,13 @@
+Package.describe({
+  summary: "Blaze configuration templates for Google OAuth.",
+  version: "1.0.0-beta.1"
+});
+
+Package.onUse(function(api) {
+  api.use('templating@1.2.13', 'client');
```

Comment:
Was there a reason for depending specifically on `1.2.13`?  We do this in `facebook-config-ui` but I don't immediately remember why.